packages/esm-commons-lib/src/components/banner-tags/patient-status-tag.component.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
rename this file to exclude the word `component`. Let it just be `patient-status-tag.test.tsx`

Reply:
@pirupius . This is done. Kindly re-review. Thanks